src/arkit.cs

```diff
@@ -509,6 +518,106 @@ interface ARSCNDebugOptions {
 		[Field ("ARSCNDebugOptionShowFeaturePoints")]
 		SCNDebugOptions ShowFeaturePoints { get; }
 	}
+
+	[iOS (11,0)]
+	[NoWatch, NoTV, NoMac]
+	[Protocol]
+	interface ARTrackable {
+		[Abstract]
+		[Export ("isTracked")]
+		bool IsTracked { get; }
+	}
+
+	[iOS (11,0)]
+	[NoWatch, NoTV, NoMac]
+	[BaseType (typeof(ARConfiguration))]
```

Comment:
there's a `[DisableDefaultCtor]` on the base type
but in this case it might be valid but...
_The abstract base class for AR session configurations._ https://developer.apple.com/documentation/arkit/arconfiguration

so there's an `[Abstract]` missing from `ARConfiguration`

Reply:
`ARConfiguration` should indeed be `[Abstract]` and `[DisableDefaultCtor]` is right here (:

```
/** Unavailable */
- (instancetype)init NS_UNAVAILABLE;
+ (instancetype)new NS_UNAVAILABLE;
```